marquee/marquee.ino

```diff
@@ -399,17 +397,18 @@ void loop() {
       if (currentTemp.length() >= 3) {
         timeSpacer = " ";
       }
-      hourMinutes += timeSpacer + currentTemp + getTempSymbol();
+      currentTime += timeSpacer + currentTemp + getTempSymbol();
     }
     if (Wide_Clock_Style == "2") {
-      hourMinutes += ":" + timeClient.getSeconds();
+      currentTime += ":" + timeClient.getSeconds();
```

Comment:
When running the double wide where you have the option to display the time in HH:mm:ss the : dots do not flash between the minutes and seconds.   We will need to update this probably to a method that returns a space or dots.

Reply:
I'll submit another PR that should handle the blink in the double wide, but I only built single display units, so I didn't verify. Also the blinking cursor could be made an option in the config, rather than the default? And, yea... Sorry about that last PR.

Struggling with the branch and submitting a PR, so try this:

    if (Wide_Clock_Style == "2") {
      currentTime += secondsIndicator(false) + timeClient.getSeconds();
      matrix.fillScreen(LOW); // show black
    }

Reply:
Oh, I have already merged your PR into the dev 2.6 branch and have made the changes there.  Please pull down the latest from 2.6 and have a look.   I made a few other minor changes.  Thanks!

Reply:
Cool. Looks good!